packages/react-next/etc/react-next.api.md

```diff
@@ -4,12 +4,66 @@
 
 ```ts
 
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { BaseButton } from 'office-ui-fabric-react/lib/Button';
+import { ButtonType } from 'office-ui-fabric-react/lib/Button';
+import { CommandBarButton } from 'office-ui-fabric-react/lib/Button';
+import { CommandButton } from 'office-ui-fabric-react/lib/Button';
+import { Button as CompatButton } from 'office-ui-fabric-react/lib/Button';
+import { Checkbox as CompatCheckbox } from 'office-ui-fabric-react/lib/Checkbox';
+import { CheckboxBase as CompatCheckboxBase } from 'office-ui-fabric-react/lib/Checkbox';
+import { Link as CompatLink } from 'office-ui-fabric-react/lib/Link';
+import { LinkBase as CompatLinkBase } from 'office-ui-fabric-react/lib/Link';
+import { Pivot as CompatPivot } from 'office-ui-fabric-react/lib/Pivot';
+import { PivotBase as CompatPivotBase } from 'office-ui-fabric-react/lib/Pivot';
+import { PivotItem as CompatPivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { PivotLinkFormat as CompatPivotLinkFormat } from 'office-ui-fabric-react/lib/Pivot';
+import { PivotLinkSize as CompatPivotLinkSize } from 'office-ui-fabric-react/lib/Pivot';
+import { Slider as CompatSlider } from 'office-ui-fabric-react/lib/Slider';
+import { SliderBase as CompatSliderBase } from 'office-ui-fabric-react/lib/Slider';
+import { Toggle as CompatToggle } from 'office-ui-fabric-react/lib/Toggle';
+import { ToggleBase as CompatToggleBase } from 'office-ui-fabric-react/lib/Toggle';
 import { ComposePreparedOptions } from '@fluentui/react-compose';
+import { CompoundButton } from 'office-ui-fabric-react/lib/Button';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { ElementType } from 'office-ui-fabric-react/lib/Button';
+import { IBaseButtonProps } from 'office-ui-fabric-react/lib/Button';
+import { IBaseButtonState } from 'office-ui-fabric-react/lib/Button';
 import { IBaseProps } from 'office-ui-fabric-react/lib/Utilities';
+import { IButton } from 'office-ui-fabric-react/lib/Button';
 import { IButtonProps } from 'office-ui-fabric-react/lib/Button';
-import { IButtonProps as IButtonProps_2 } from 'office-ui-fabric-react/lib/components/Button/Button.types';
+import { IButtonProps as IButtonProps_2 } from '@fluentui/react-next/lib/CompatButton';

```

Comment:
Anywhere you're getting a _2 or _3 means that you're exporting the same name twice with different definitions, which is going to cause problems for consumers.

Reply:
Resolved!